1-resilient-messaging/docker-compose.yml

```diff
@@ -47,4 +47,4 @@ services:
     networks:
       - default
     ports:
-      - "3000:80"
+      - "3001:80"
```

Comment:
@dgafka when I did docker-compose up with port 3000 I received error:
`Error response from daemon: driver failed programming external connectivity on endpoint 1-resilient-messaging-ecotone_pulse-1 (4f6b0c987495941a3eb3b26da807a15a72d0eb8c087a6c19c7867e21d7267557): Bind for 0.0.0.0:3000 failed: port is already allocated`

Reply:
When I changed the port fro 3000 to 3001 after startup I can access Ecotone pulse in my browser at localhost:3001

Reply:
Jak już odpowiadamy po polsku a i tak repo jest opisane po polsku, to tak będę kontynuował :P 

Z tego wynika, że masz już ten port zarezerwowany i coś Ci na nim chodzi.
To samo może być z bazą danych i RabbitMQ, które się bindują do host'a. 

To może warto zmienić wszystkie bindowania na porty 4000 i wyżej, tak aby nie wpaść w przypadkowe konflikty?

Reply:
Ok, przetestuję konfigurację z portami >4000 i dodam do tego pull request